### PR TITLE
Update JUnit reporter to add time attribute to testsuites and testcas…

### PR DESCRIPTION
**Pull request by dopsonbr**
…e elements.

The time attribute is required for the testsuites and testcase elements per the
JUnit spec.

### CODE REVIEW

File: lib/reporters/junit/index.js [hunk 1]
```diff
@@ -17,8 +17,12 @@ JunitReporter = function (newman, reporterOptions) {
         var report = _.get(newman, 'summary.run.executions'),
             collection = _.get(newman, 'summary.collection'),
             cache,
-            root;
+            root,
+            testSuitesExecutionTime,
+            executionTime;
 
+        testSuitesExecutionTime = 0;
```
Comment (kunagpal) [inline lib/reporters/junit/index.js]: Minor change: The initializations in these two lines can be shifted above.

Comment (dopsonbr) [inline lib/reporters/junit/index.js]: Thanks. Ill make the changes and merge develop into this branch today

**Comment by dopsonbr** [comment]
@shamasis I agree that the test suites execution time should be the total execution not just the summation of the response times. However, it appears the current code is setting the test suite time to the response time and the test suites execution time should be the summation of each test suite.

I wanted to make the changes as small as possible because this is my first open source contribution and am still learning the rules of the road.

File: lib/reporters/junit/index.js (hunk 1 above)
Comment (dopsonbr) [inline lib/reporters/junit/index.js]: The changes have been made and branch updated.

**Comment by kunagpal** [comment]
This change is now available in Newman v3.6.1, do check it out 😄

**Pull request merged**